src/pages/api/getSummaryTxMetadata/[pollId].ts

```diff
@@ -95,9 +97,52 @@ export default async function getSummaryTxMetadata(
     // Iterate through poll_transaction IDs and get associated TX ID
     const txIds = await pollTransactionsDto(pollTransactionIds);
 
+    const yesVotes = pollVotes.filter((vote) => vote.vote === 'yes').length;
+    const noVotes = pollVotes.filter((vote) => vote.vote === 'no').length;
+    const abstainVotes = pollVotes.filter(
+      (vote) => vote.vote === 'abstain',
+    ).length;
+
+    const workshops = await workshopsDto();
+
+    const activeWorkshops = workshops.filter(
+      (workshop) => workshop.active_voter_id,
+    );
+    const activeVoterCount = activeWorkshops.length;
+
+    const percentage =
+      Math.round((yesVotes / (activeVoterCount - abstainVotes)) * 100) || 0;
+
+    const approvalString = `Approval Result: ${percentage}%`;
+    const yesString = `Yes Votes: ${yesVotes}`;
+    const noString = `No Votes: ${noVotes}`;
+    const abstainString = `Abstain Votes: ${abstainVotes}`;
+    const totalString = `Total Eligible Voters: ${activeVoterCount}`;
+
+    const metadata = [
+      'Text Hash:',
+      poll.hashedText,
+      approvalString,
+      yesString,
+      noString,
+      abstainString,
+      totalString,
+      'Delegate Signature Transaction Hashes:',
+      ...txIds,
+    ];
+
+    // make sure metadata is under 16kb
+    const bytes = getBytesOfArray(metadata);
+    if (bytes > 16000) {
```

Comment:
Need to leave a bit of room for the rest of the TX info since total TX size must be below 16kb.
```suggestion
    if (bytes > 15000) {
```